File: sample01/src/main/java/com/jworkflow/sample01/steps/Hello.java
package com.jworkflow.sample01.steps;

import com.jworkflow.kernel.interfaces.StepBody;
import com.jworkflow.kernel.models.*;

public class Hello implements StepBody {

    @Override
    public ExecutionResult run(StepExecutionContext context) {
        System.out.println("Hello world");
        return ExecutionResult.next();
    }    
}

File: sample01-kotlin/src/main/java/net/jworkflow/sample01/steps/Hello.kt
package net.jworkflow.sample01.steps
import net.jworkflow.kernel.interfaces.StepBody
import net.jworkflow.kernel.models.*

class Hello:StepBody {
    @Override
    fun run(context:StepExecutionContext?):ExecutionResult? {
        System.out.println("Hello world")
        return ExecutionResult.next()
    }
}